Makefile

```diff
@@ -168,7 +168,7 @@ manager: ## Build manager binary.
 
 .PHONY: clusterctl
 clusterctl: ## Build clusterctl binary.
```

Comment:
Can we remove this target entirely?

Reply:
And everywhere we invoke `bin/clusterctl`, change it to `clusterctl` (must be on `$PATH`; require users to install it from CAPI/brew/etc)?

Reply:
Once we have a release and means to install clusterctl/any other tool, that'd be fine. I would like to keep this target for now while we develop locally